components/listing/listing-card.tsx

```diff
@@ -120,40 +122,39 @@ const SVGHeart = ({ width, height, color }) => (
 );
 
 interface ListingCard {
-  data: {
-    location: string;
-    title: string;
-    description: string;
-    price: number;
-    rate: number;
-    reviews: number;
-    imageUrl: React.ComponentProps<typeof Image>["src"];
-  };
+  data: TToursResponse;
 }
 
 const ListingCard: React.FC<ListingCard> = ({ data }) => {
   const [like, setLike] = useState(false);
   return (
     <CardContainer>
       <ImageContainer>
-        <Link href={`/tours/${data.title}`}>
-          <Image src={data.imageUrl} alt="Scenic Forest" layout="fill" />
+        <Link href={`/tours/${data.tour_slug}`}>
+          <Image
+            src={data.tour_banner_image}
+            alt="Scenic Forest"
+            loading="lazy"
+            layout="fill"
+          />
         </Link>
         <ActionButton onClick={() => setLike(!like)}>
           <SVGHeart width={20} height={20} color={like ? "red" : "white"} />
         </ActionButton>
       </ImageContainer>
-      <Link href={`/tours/${data.title}`}>
+      <Link href={`/tours/${data.tour_slug}`}>
         <DetailsContainer>
-          <p className="location">{data.location}</p>
+          <p className="location">{"none"}</p>
```

Comment:
we need to add this on the API. but what is the purpose of `none`?

Reply:
For placeholder purposes since nageerror yung code, pero typescript error pala yung issue.